scripts/pca_variant_filter.py

```diff
@@ -74,313 +72,369 @@ def determine_pca_variants(
+                )
 
-        logger.info("Densifying filtered CCDG %s VDS...", data_type)
-        mt = hl.vds.to_dense_mt(vds)
-        if adj_only:
-            mt = filter_to_adj(mt)
+            logger.info("Densifying filtered CCDG %s VDS...", data_type)
+            mt = hl.vds.to_dense_mt(vds)
+            if adj_only:
+                mt = filter_to_adj(mt)
+
+            annotation_expr = {
+                f"ccdg_{data_type}_was_split": mt.was_split,
+                f"ccdg_{data_type}_AC": hl.agg.sum(mt.GT.n_alt_alleles()),
+                f"ccdg_{data_type}_AN": hl.agg.count_where(hl.is_defined(mt.GT)) * 2,
+            }
+
+            if min_inbreeding_coeff_threshold is not None:
+                annotation_expr[
+                    f"ccdg_{data_type}_site_inbreeding_coeff"
+                ] = bi_allelic_site_inbreeding_expr(mt.GT)
+            if min_hardy_weinberg_threshold is not None:
+                annotation_expr[f"ccdg_{data_type}_hwe"] = hl.agg.hardy_weinberg_test(
+                    mt.GT
+                )
 
-        annotation_expr = {
-            f"ccdg_{data_type}_was_split": mt.was_split,
-            f"ccdg_{data_type}_AC": hl.agg.sum(mt.GT.n_alt_alleles()),
-            f"ccdg_{data_type}_AN": hl.agg.count_where(hl.is_defined(mt.GT)) * 2,
-        }
+            mt = mt.annotate_rows(**annotation_expr)
+            ht = mt.rows().checkpoint(
+                get_ccdg_results_path(
+                    data_type=data_type,
+                    mt=False,
+                    result=f"pre_filtered_variants_interval{INTERVAL_DP}x{flag}",
+                ),
+                overwrite=(not read_per_dataset_checkpoint_if_exists),
+                _read_if_exists=read_per_dataset_checkpoint_if_exists,
+            )
 
-        if min_inbreeding_coeff_threshold is not None:
-            annotation_expr[
-                f"ccdg_{data_type}_site_inbreeding_coeff"
-            ] = bi_allelic_site_inbreeding_expr(mt.GT)
-        if min_hardy_weinberg_threshold is not None:
-            annotation_expr[f"ccdg_{data_type}_hwe"] = hl.agg.hardy_weinberg_test(mt.GT)
+            return ht
 
-        mt = mt.annotate_rows(**annotation_expr)
-        ht = mt.rows().checkpoint(
-            get_ccdg_results_path(
-                data_type=data_type, mt=False, result=f"pre_filtered_variants_interval{INTERVAL_DP}x"
-            ),
-            overwrite=overwrite,
-            _read_if_exists=(not overwrite),
+        logger.info(
+            "Creating Table with joint gnomAD v3.1.2 and CCDG genome allele frequencies and callrate...",
+        )
+        ccdg_genomes_ht = _initial_filter("genomes")
+        ccdg_exomes_ht = _initial_filter("exomes")
+        ht = ccdg_exomes_ht.join(ccdg_genomes_ht, how="inner")
+        ht = ht.annotate(**gnomad_ht[ht.key], **ukbb_ht[ht.key])
+        ht = ht.annotate(
+            # joint_biallelic=~ht.ccdg_genomes_was_split & ~ht.gnomad_was_split,
```

Comment:
Just remove if this is not what was used. You will want to modify the docstring and the script argument
```suggestion
            # joint_biallelic=~ht.ccdg_genomes_was_split & ~ht.gnomad_was_split,
```